rendering/mmd_pmd.ksy

```diff
@@ -0,0 +1,371 @@
+    repeat-expr: face_vertex_count / 3
+  - id: material_count
+    type: u4
+  - id: materials
+    type: material
+    repeat: expr
+    repeat-expr: material_count
+  - id: bone_count
+    type: u2
+  - id: bones
+    type: bone
+    repeat: expr
+    repeat-expr: bone_count
+  - id: ik_count
+    type: u2
+  - id: iks
+    type: ik
+    repeat: expr
+    repeat-expr: ik_count
+  - id: morph_count
+    type: u2
+  - id: morphs
+    type: morph
+    repeat: expr
+    repeat-expr: morph_count
+  - id: morph_frame_count
+    type: u1
+  - id: morph_frames
+    type: morph_frame
+    repeat: expr
+    repeat-expr: morph_frame_count
+  - id: bone_frame_name_count
+    type: u1
+  - id: bone_frame_names
+    type: bone_frame_name
+    repeat: expr
+    repeat-expr: bone_frame_name_count
+  - id: bone_frame_count
+    type: u4
+  - id: bone_frames
+    type: bone_frame
+    repeat: expr
+    repeat-expr: bone_frame_count
+  - id: english_header
+    type: english_header
+  - id: english_bone_names
+    type: english_bone_name
+    repeat: expr
+    repeat-expr: bone_count
+    if: english_header.compatibility > 0
+  - id: english_morph_names
+    type: english_morph_name
+    repeat: expr
+    repeat-expr: morph_count - 1
+    if: english_header.compatibility > 0
+  - id: english_bone_frame_names
+    type: english_bone_frame_name
+    repeat: expr
+    repeat-expr: bone_frame_name_count
+    if: english_header.compatibility > 0
+  - id: toon_textures
+    type: toon_texture
+    repeat: expr
+    repeat-expr: 10
+  - id: rigid_body_count
+    type: u4
+  - id: rigid_bodies
+    type: rigid_body
+    repeat: expr
+    repeat-expr: rigid_body_count
+  - id: constraint_count
+    type: u4
+  - id: constraints
+    type: constraint
+    repeat: expr
+    repeat-expr: constraint_count
+
+types:
+
+  header:
+    seq:
+      - id: magic
+        contents: 'Pmd'
+      - id: version
+        type: u4
+      - id: model_name
+        type: strz
+        size: 20
+      - id: comment
+        type: strz
+        size: 256
+
+  vertex:
+    seq:
+      - id: position
+        type: f4_3
+      - id: normal
+        type: f4_3
+      - id: uv
+        type: f4_2
+      - id: skin_indices
+        type: u2_2
+      - id: skin_weights
+        type: u1
+      - id: edge_flag
+        type: u1
+
+  face:
+    seq:
+      - id: indices
+        type: u2_3
+
+  material:
+    seq:
+      - id: diffuse
+        type: f4_4
+      - id: shininess
+        type: f4
+      - id: specular
+        type: f4_3
+      - id: ambient
+        type: f4_3
+      - id: toon_index
+        type: u1
+      - id: edge_flag
+        type: u1
+      - id: face_vertex_count
+        type: u4
+      - id: file_name
+        type: strz
+        size: 20
+
+  bone:
+    seq:
+      - id: name
+        type: strz
+        size: 20
+      - id: parent_index
+        type: u2
+      - id: tail_index
+        type: u2
+      - id: type
+        type: u1
+      - id: ik_index
```

Comment:
probably can be connected to the ik via an instance.

Reply:
Nice idea

Reply:
It could just be the web UI, but I get "Call stack: undefined RangeError: Maximum call stack size exceeded" trying to add even just:

```yaml
      parent:
        value: '_parent.bones[parent_index]'
      tail:
        value: '_parent.bones[tail_index]'
```

Whereas just this is fine:
```yaml
      parent:
        value: '_parent.bones[parent_index]'
```